python/envconfig.json

```diff
@@ -0,0 +1,24 @@
+[{
+    "version": "1.0.0", 
+    "runtimeVersion" : "1.0.0", 
```

Comment:
We have Python 2 & 3 - so this is incorrect

Reply:
chnaged